stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -345,6 +375,14 @@ public final class _ExecutionContext {
       self.gpuDeviceNamePrefix = nil
     }
 
+    if tpuCount > 0 {
+      // According to server def generated when you set
+      // SWIFT_TENSORFLOW_SERVER_ADDRESS, the TPUs will all be on task 1.
+      self.tpuDeviceNamePrefix = "/job:localhost/replica:0/task:1/device:TPU:"
+    } else {
+      self.tpuDeviceNamePrefix = nil
```

Comment:
There's no need for this `else` block. Optional members are `nil` by default.

Reply:
I thought the same, but this is a let struct member, so this is needed or it complains that not all paths initialize tpuDeviceNamePrefix.